content/en/logs/log_configuration/archives.md

```diff
@@ -58,7 +58,7 @@ If not already configured, set up the [AWS integration][1] for the AWS account t
 
 Set up the [Azure integration][1] within the subscription that holds your new storage account, if you haven't already. This involves [creating an app registration that Datadog can use][2] to integrate with.
 
-**Note:** Archiving to Azure ChinaCloud is not supported.
+**Note:** Archiving to Azure ChinaCloud, GermanyCloud and GovCloud are not supported. We only support archiving logs to Azure public cloud.
```

Comment:
```suggestion
**Note:** Archiving to Azure ChinaCloud, GermanyCloud, and GovCloud is not supported.
```

Reply:
Thanks for feedback. I have committed your suggestion.